client/web/compose/src/components/PageBlocks/RecordRevisionsBase.vue

```diff
@@ -181,6 +187,14 @@ export default {
     noRecord () {
       return !this.record
     },
+
+    isPageBuilder () {
```

Comment:
Why introduce the two new computeds if you can just check if `this.record.recordID === NoID`